tools/cli.py

```diff
@@ -195,6 +196,12 @@ def Main() -> None:
             args=[
                 ('name', 'The name of the bucket.', None),
             ])
+  AddParser('aws', aws_subparsers, 'uploadtobucket',
+            'Upload a file to an S3 bucket.',
+            args=[
+                ('bucket', 'The name of the bucket.', None),
+                ('filename', 'Local file name.', None),
```

Comment:
I think we should name this argument "filepath", since the `put` function is actually expecting a full path rather than a file name

Reply:
Good point, done.